doc/classes/Geometry3D.xml

```diff
@@ -73,6 +73,16 @@
 				Given the two 3D segments ([param p1], [param p2]) and ([param q1], [param q2]), finds those two points on the two segments that are closest to each other. Returns a [PackedVector3Array] that contains this point on ([param p1], [param p2]) as well the accompanying point on ([param q1], [param q2]).
 			</description>
 		</method>
+		<method name="get_triangle_barycentric_coords">
+			<return type="Vector3" />
+			<param index="0" name="point" type="Vector3" />
+			<param index="1" name="v0" type="Vector3" />
+			<param index="2" name="v1" type="Vector3" />
+			<param index="3" name="v2" type="Vector3" />
+			<description>
+				Returns the barycentric coordinates for a given point inside a triangle.
```

Comment:
It would be useful to describe what barycentric coordinates *are*, and what you might want to use them for. Linking to a Wikipedia page or similar should be a good start, but ideally the class reference should describe potential use cases in a game (e.g. rotating decals based when creating decals near surfaces).

Reply:
Added, thanks.

Reply:
To answer your question, the two are unrelated. This function can be used in many more scenarios than just getting data from collision. An example in the Docs might be nice, but I'm not sure it should be in the method description.